lib/cloudshaper/cli.rb

```diff
@@ -77,7 +78,8 @@ def version
 
     private
 
-    def load_stack(stack, _options, pull = true)
+    def load_stack(stack, pull = true)
```

Comment:
Named arguments is a lot nicer: `def load_stack(stack, pull: true)`
This allows you to call this method like this `load_stack(name, pull: false)`. 

This makes the intent of the boolean argument a lot clearer.